riverdriver/riverpgxv5/migration/main/005_river_migration_add_line.up.sql

```diff
@@ -15,4 +15,35 @@ INSERT INTO river_migration
 SELECT created_at, 'main', version
 FROM river_migration_old;
 
-DROP TABLE river_migration_old;
\ No newline at end of file
+DROP TABLE river_migration_old;
+
+--
+-- Create `river_client` and derivative.
+--
+-- This feature hasn't quite yet been implemented, but we're taking advantage of
+-- the migration to add the schema early so that we can add it later without an
+-- additional migration.
+--
+
+CREATE UNLOGGED TABLE river_client (
+    name text PRIMARY KEY NOT NULL,
+    created_at timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    metadata jsonb NOT NULL DEFAULT '{}' ::jsonb,
+    paused_at timestamptz,
+    updated_at timestamptz NOT NULL,
+    CONSTRAINT name_length CHECK (char_length(name) > 0 AND char_length(name) < 128)
+);
+
+-- Differs from `river_queue` in that it tracks the queue state for a particular
+-- active client.
+CREATE UNLOGGED TABLE river_client_queue (
+    river_client_name text NOT NULL REFERENCES river_client (name) ON DELETE CASCADE,
+    name text NOT NULL,
+    created_at timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
huh, I've always used and seen `DEFAULT now()` and never seen this form. Maybe we want to stick with `now()` for consistency within the project since we use it elsewhere?

Reply:
Yeah, I'll change them back to `now()`.

One thing I learned while trying to implement SQLite is that `now()` is actually a Postgres thing while `CURRENT_TIMESTAMP` is a standard that works everywhere. It's not a problem for now, but if we bring SQLite in we may want to switch to `CURRENT_TIMESTAMP` everywhere for consistency since SQLite will need it that way anyway.